apis/nucleus/src/components/listbox/ListBox.jsx

```diff
@@ -112,8 +113,21 @@ export default function ListBox({
     [layout, pages]
   );
 
+  const applyValueFilter = (logPaginos, filterFunc) => {
```

Comment:
Rename `logPaginos` to receivedPages?

Reply:
Yeah, and the correct spelling should also be `losPaginos` 😄 .
My variable naming clearly needs some help here. Thanks!